src/components/include/cos_kernel_api.h

```diff
@@ -30,7 +30,7 @@ struct cos_meminfo {
 /* Component captbl/pgtbl allocation information */
 struct cos_compinfo {
 	/* capabilities to higher-order capability tables (or -1) */
-	capid_t pgtbl_cap, captbl_cap, comp_cap;
+	capid_t pgtbl_cap, captbl_cap, comp_cap, local_pgtbl_cap;
```

Comment:
I'm probably a bit rusty here -- why we need a global_pgtbl_cap in the compinfo? What are the valid use cases? Bear with me if I'm asking stupid questions.